components/brave_wallet_ui/page/async/wallet_page_async_handler.ts

```diff
@@ -299,9 +302,8 @@ handler.on(WalletPageActions.getNFTMetadata.getType(), async (store, payload: Br
   store.dispatch(WalletPageActions.setIsFetchingNFTMetadata(false))
 })
 
-handler.on(WalletPageActions.onOnboardingShown.getType(), async (store: Store) => {
-  const braveWalletService = getWalletPageApiProxy().braveWalletService
-  await braveWalletService.onOnboardingShown()
+handler.on(WalletPageActions.reportOnboardingAction.getType(), async (store: Store, payload: OnboardingAction) => {
```

Comment:
We don't need this action/reducer anymore. The Api proxy can now be accessed from React components directly using the `useApiProxy` hook.